### PR TITLE
Project Ideas dropdown section added

### LINKED ISSUE #220: Adding a project ideas section
## Proposal

I want to add a project ideas section in the navbar "more" section. I will provide a link to a website that displays various project ideas for college students.

I want to work on this issue as a participant of GSSOC'21, please assign it to me.

### PR DESCRIPTION
**Pull request by gayatri517**
# Description
I have added a project ideas section in the more section of the navbar, where a link to a website is provided listing various project ideas.
Fixes #220 

## List any dependencies that are required for this change 
None

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?
I have verified the link attached. It is working perfectly.

## UI /UX changes
   Attach gif or screenshot for changes.
   - Before : 
before

- After : 

project Idea

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by gayatri517** [comment]
@harikesh409 Kindly review the PR.
Thank you.

**Comment by gayatri517** [comment]
@harikesh409 kindly review my PR.
Thank you

**Review by harikesh409 — CHANGES_REQUESTED** [review]
As it is an external website instead of adding in the navbar add it in the Features section in footer.

**Comment by gayatri517** [comment]
@harikesh409 I have made the changes. Kindly review it.
Thank you.

File: index.html [hunk 1]
```diff
@@ -90,7 +90,7 @@
                                 to Project</a>
 
                             <a class="dropdown-item" href="practicedsa.html">DSA Problems</a>
-
+
```
Comment (harikesh409) [inline index.html:93]: Remove blank line.

**Review by harikesh409 — CHANGES_REQUESTED** [review]

**Comment by gayatri517** [comment]
@harikesh409 I have made the changes. Kindly review.
Thank you

**Review by harikesh409 — APPROVED** [review]

**Review by harikesh409 — CHANGES_REQUESTED** [review]
As it is an external link instead of keeping it in the navbar move that to the footer under `Features` section.

**Comment by gayatri517** [comment]
@harikesh409 I have changed the link from the navbar to the footer section
pic1
pic2

**Comment by harikesh409** [comment]
> @harikesh409 I have changed the link from the navbar to the footer section
> pic1
> pic2

Push your commits. @gayatri517

**Comment by gayatri517** [comment]
@harikesh409 I have done the changes. Kindly review now.
Thanks

**Pull request merged**